src/components/Vocal.js

```diff
@@ -116,24 +129,50 @@ const Vocal = ({
 	}
 
 	const _renderDefault = () => (
-		<div
+		<button
 			data-testid="__vocal-root__"
+			ref={buttonRef}
 			role="button"
 			aria-label={ariaLabel}
 			tabIndex={tabIndex}
-			style={className ? null : { width: 24, height: 24, cursor: !isListening ? 'pointer' : null, ...style }}
+			style={
+				className
+					? null
+					: {
+							width: 24,
+							height: 24,
+							background: 'none',
+							border: 'none',
+							padding: 0,
+							cursor: !isListening ? 'pointer' : 'default',
+							...style,
+					  }
+			}
 			className={className}
+			onFocus={_onFocus}
+			onBlur={_onBlur}
+			onClick={_onClick}
 		>
 			<Icon isActive={isListening} iconColor="#aaa" />
-		</div>
+		</button>
 	)
 
-	return (
-		SpeechRecognitionWrapper.isSupported &&
-		cloneElement(isValidElement(children) ? children : _renderDefault(), {
-			...(!isListening && { onClick: _onClick }),
-		})
-	)
+	const _renderChildren = (children) => {
+		if (SpeechRecognitionWrapper.isSupported) {
+			if (isFunc(children)) {
+				return children(startRecognition, stopRecognition)
+			} else if (isValidElement(children)) {
+				return cloneElement(children, {
+					...(!isListening && { onClick: _onClick }),
```

Comment:
Fonctionne dans assez peu de cas, il faut un enfant unique qui a un onClick ! La doc pourrait être plus claire là-dessus. En vrai, c'est assez limité pour que je pense que ça ne vaille pas le coup d'avoir ce cas d'usage.

Reply:
Agree, this is for compatibility purpose